addon/components/date-time-picker.js

```diff
@@ -35,6 +35,16 @@ export default Component.extend({
    */
   datePickerClasses: '',
 
+  /**
+   * The placeholder for the date button, if no date is selected.
+   *
+   * @attribute datePlaceholder
+   * @type {String}
+   * @default 'Select Date...'
+   * @public
+   */
+  datePlaceholder: 'Select Date...',
```

Comment:
Please leave this `null` by default, so that we use the default date/time placeholders. This way, you only need to overwrite the defaults in one place (the date/time component itself), not here as well! The same for `timePlaceholder`.